frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -24,21 +32,75 @@ const Input = <T extends FieldValues>({
     name,
     placeholder = '',
     type = 'text',
+    isDisabled = false,
+    adornmentText = '',
 }: Properties<T>): JSX.Element => {
     const { field } = useFormController({ name, control });
 
     const error = errors[name]?.message;
     const hasError = Boolean(error);
 
+    let adornment = null;
+    if (type === 'search') {
+        const iconAdornmentStyles = getValidClassNames(
+            styles['MuiInputAdornment-root'],
+            styles.adornmentIcon,
+        );
+        adornment = (
+            <InputAdornment position="start" className={iconAdornmentStyles}>
+                <Search />
+            </InputAdornment>
+        );
+    }
+    if (adornmentText) {
+        const textAdornmentStyles = getValidClassNames(
+            styles['MuiInputAdornment-root'],
+            styles.adornmentText,
+        );
+        adornment = (
+            <InputAdornment position="start" className={textAdornmentStyles}>
+                {adornmentText}
+            </InputAdornment>
+        );
+    }
+
+    const inputLabelStyles = getValidClassNames(
+        styles.label,
+        isDisabled && styles.labelDisabled,
+    );
+    const textFieldRootStyles = getValidClassNames(styles.root);
+    const muiInputStyles = getValidClassNames(
+        styles.inputWrapper,
+        isDisabled && styles.inputDisabled,
+        hasError && styles.hasError,
+    );
+    const htmlInputStyles = getValidClassNames(
+        styles.input,
+        type === 'search' && styles.inputPaddingSearch,
+        adornmentText && styles.inputPaddingTextAdornsment,
+    );
+    const helperTextStyles = getValidClassNames(
+        styles.helperText,
+        hasError && styles.hasError,
+    );
+
     return (
-        <label>
-            <span>{label}</span>
+        <label className={styles.wrapper}>
```

Comment:
we don't need label, we just need input component

Reply:
So should I delete from input component all labels and return only TextField. Like this:
`return (
            <TextField
                {...field}
                type={type}
                placeholder={placeholder}
                error={hasError}
                helperText={(error as string) || ' '}
                className={textFieldRootStyles}
                InputProps={{
                    className: muiInputStyles,
                    disabled: isDisabled,
                    startAdornment: adornment,
                }}
                inputProps={{ className: htmlInputStyles }}
                FormHelperTextProps={{ className: helperTextStyles }}
            />
    );`
and remobe label from Props of component?